Formula/l/logalize.rb

```diff
@@ -0,0 +1,39 @@
+class Logalize < Formula
+  desc "Fast and extensible log colorizer. Alternative to ccze"
+  homepage "https://github.com/deponian/logalize"
+  url "https://github.com/deponian/logalize.git",
+    tag:      "v0.4.1",
+    revision: "2d74f36f36037d1a09371f044a85b2b8f192905f"
```

Comment:
```suggestion
  url "https://github.com/deponian/logalize/archive/refs/tags/v0.4.1.tar.gz"
  sha256 "xxx"
```

Reply:
The application needs commit hash during the building stage, so I need to use tag-revision approach here